src/client/public/locale/de/common.json

```
@@ -177,6 +177,7 @@
   "importedData": "Importierte Daten",
   "boreholesImported": "Bohrungen wurden importiert",
   "boreholesImportError": "Es konnten keine Bohrungen importiert werden! Bitte stellen Sie sicher, dass die hochgeladene CSV-UTF8-Datei das korrekte Format hat.",
+  "boreholesImportLongRunning": "Der Import läuft im Hintergrund weiter, dies kann einige Minuten dauern. Nach Abschluss werden die Bohrungen in der Liste sichtbar.",
```

PP: Oder so ähnlich, dit dann für die anderen Sprachen.

```suggestion
  "boreholesImportLongRunning": "Der Import läuft im Hintergrund weiter, dies kann einige Minuten dauern. Sofern die zu importierten Daten korrekt sind und es keine Duplikate gibt, werden die Bohrungen nach Abschluss in der Liste sichtbar.",
```

---

Ich verstehe dein Vorschlag. Jedoch gehen wir davon aus, dass allfällige Validierungsfehler und Duplikate innert sehr kurzer Zeit festgestellt werden (das passiert vor den zeitfressenden requests wegen der Location). Das heisst wir gehen davon aus, dass ein Upload der länger als 30 s dauert durchlaufen wird. (Sofern beim S3 upload oder beim db commit etc. nichts mehr schief läuft). Aber es ist keine ideale Lösung und sollte der Upload doch failed ist es nicht ersichtlich für den User. Macht dann die originale Formulierung Sinn?

---

Für den heutigen Release wären die Changes dieses PRs sinnvoll. Die Textanpassung kann nachgeliefert werden :-).